src/wp-includes/block-bindings/sources/pattern.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * The "pattern" source for the Block Bindings API. This source is used by the
+ * Pattern Overrides.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ */
+function pattern_source_callback( $source_attrs, $block_instance, $attribute_name ) {
+	if ( ! _wp_array_get( $block_instance->attributes, array( 'metadata', 'id' ), false ) ) {
+		return null;
+	}
+	$block_id = $block_instance->attributes['metadata']['id'];
+	return _wp_array_get( $block_instance->context, array( 'pattern/overrides', $block_id, $attribute_name ), null );
+}
+
+register_block_bindings_source(
```

Comment:
Can we wrap these in `init` action maybe?

Reply:
My omission, it was supposed to be wrapped with `init` as it's already done in GB. Done in 0bec8330a7102284a747458b37b3960cab4321a6